src/bot/lib/modifiers/parse-user.js

```diff
@@ -1,10 +1,10 @@
 export function parseUser() {
   return (incomingMessage, messageContext) => ({
-    ...messageContext,
```

Comment:
Without reusing incoming messageContext, this will erase default value again

Reply:
`registerCommand` does the merge now. It let us avoid returning `messageContext` if we don't use it. See https://github.com/mindsers/sanpe-bot/pull/17/files#diff-f4fd88ee249fe934c063de2e756fd76d0a610ab3ad0068edb1dbbe870344b387R27

It only moved the code in a unique place.